modules/i18n/src/translations/en-US/portals/custom-auth-connection.ts

```diff
@@ -136,6 +136,14 @@ export const customAuthentication: customAuthenticationNS = {
             },
             subTitle: "Register externally implemented authentication service.",
             title: "Custom Authentication"
+        },
+        editPage: {
+            generalTab: {
+                iconUrl: {
+                    hint: "An image URL for the application. If this is not provided, we will display a generated" +
+                    "thumbnail instead. Recommended size: 200x200 pixels."
```

Comment:
Shall we use the same hint in the connection logo field?

```suggestion
                    hint: "A URL for the image of the connection for display purposes. If not provided a generated thumbnail will be displayed. Recommended size is 200x200 pixels."
```

Reply:
This is updated with the commit e55e8fe89c1b6eb62f5239b934368a7b32477b36